presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -469,14 +465,13 @@ private void planDistribution(PlanRoot plan)
         stateMachine.beginDistributedPlanning();
```

Comment:
Note the method name, comment and state machine are still using the word `distributed plan`. Maybe left a TODO so we can cleanup them later?

Also, the comment `time distribution planning` looks interesting. Are we expecting distributed planning (i.e. creates all the split sources) taking some significant amount of time?

Reply:
I'm going to remove the distributed planning time as a separate commit / PR.